assets/css/sensei-course-theme/blocks/course-progress-bar.scss

```diff
@@ -1,7 +1,6 @@
 .sensei-course-theme-course-progress-bar {
 	height: var(--header-progress-bar-height, 4px);
 	background-color: var(--border-color);
-	margin: 0 !important;
```

Comment:
Revert it to see how it overwrites margins set in global styles.
And keep it removed to see the problem for users with existing template.